README.md

```diff
@@ -12,3 +12,10 @@ pip install -e .
 Examples of analysis repositories making use of `topcoffea`:
 * [`topeft`](https://github.com/TopEFT/topeft): EFT analyses in the top sector. 
 * [`ewkcoffea`](): Multi boson analyses. 
+
```

Comment:
I think it would make more sense for this description to go in the Readme in the TauSF directory.

Reply:
@jlawrenc could you move this out of the main README and into the TauSF one? I don't think this PR needs to touch the main README.

Reply:
I'm going to go ahead and move this myself, just so we can get the PR merged.